cfme/tests/cloud_infra_common/test_vm_instance_analysis.py

```diff
@@ -329,6 +367,75 @@ def test_ssa_template(local_setup_provider, provider, soft_assert, vm_analysis_p
         soft_assert(c_fs_drivers != '0', "fs drivers: '{}' != '0'".format(c_fs_drivers))
 
 
+@pytest.mark.tier(2)
+@pytest.mark.long_running
+def test_ssa_schedule(ssa_vm, schedule_ssa, soft_assert, appliance):
+    """ Tests SSA can be performed and returns sane results
+
+    Metadata:
+        test_flag: vm_analysis
+    """
+    e_users = None
+    e_groups = None
+    e_packages = None
+    e_services = None
+    e_os_type = ssa_vm.system_type['os_type']
+
+    if ssa_vm.system_type != WINDOWS:
+        e_users = ssa_vm.ssh.run_command("cat /etc/passwd | wc -l").output.strip('\n')
+        e_groups = ssa_vm.ssh.run_command("cat /etc/group | wc -l").output.strip('\n')
+        e_packages = ssa_vm.ssh.run_command(
+            ssa_vm.system_type['package-number']).output.strip('\n')
+        e_services = ssa_vm.ssh.run_command(
+            ssa_vm.system_type['services-number']).output.strip('\n')
+
+    logger.info("Expecting to have {} users, {} groups, {} packages and {} services", e_users,
+                e_groups, e_packages, e_services)
+    # Check release and quadicon
+    quadicon_os_icon = ssa_vm.find_quadicon().data['os']
+    view = navigate_to(ssa_vm, 'Details')
+    details_os_icon = view.entities.summary('Properties').get_text_of('Operating System')
+    logger.info("Icons: %s, %s", details_os_icon, quadicon_os_icon)
+    c_lastanalyzed = ssa_vm.last_analysed
+    c_users = view.entities.summary('Security').get_text_of('Users')
+    c_groups = view.entities.summary('Security').get_text_of('Groups')
+    c_packages = 0
+    c_services = 0
+    if ssa_vm.system_type != WINDOWS:
+        c_packages = view.entities.summary('Configuration').get_text_of('Packages')
+        c_services = view.entities.summary('Configuration').get_text_of('Init Processes')
+
+    logger.info("SSA shows {} users, {} groups {} packages and {} services", c_users, c_groups,
+                c_packages, c_services)
+
+    soft_assert(c_lastanalyzed != 'Never', "Last Analyzed is set to Never")
```

Comment:
**REQUIRED**
I think here should be usual `assert`, because further checks don't make sense if a vm was not analyzed.